sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/plugin/agent/enhance/AbstractClassLoaderInterceptor.java

```diff
@@ -0,0 +1,44 @@
+package io.sermant.core.plugin.agent.enhance;
```

Comment:
Add copy right and licence for all new class files. You can refer to https://github.com/sermant-io/Sermant/blob/6b2582d346fef1ad358c1b408697b1c8749d6ff7/sermant-backend/src/main/java/io/sermant/backend/handler/config/ServiceRegistryPluginHandler.java